common/logisticspipes/blocks/powertile/LogisticsPowerJunctionTileEntity.java

```diff
@@ -266,13 +268,13 @@ public void onChunkUnload() {
 
 	@Override
 	@CCCommand(description = "Returns the currently stored power")
-	public int getPowerLevel() {
+	public int getAvailablePower() {
```

Comment:
Don't do this. This is like a public API for Computer Mods (the CCCommand annotation). This will break scripts people have been using for years.